inputfiles/overridingTypes.json

```diff
@@ -129,6 +129,18 @@
                     "property": {
                         "onerror": {
                             "override-type": "OnErrorEventHandler"
+                        },
+                        "ontouchcancel": {
+                            "override-type": "((this: Window, ev: TouchEvent) => any) | null | undefined"
```

Comment:
Try replacing this line with `nullable: 1`.

Reply:
Of course, why didn't I think that `nullable: 1` would change a `nullable` type into a `nullable | undefined` type...